src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,102 @@
-import React from 'react';
+import { useState } from 'react';
 import './NewCommentForm.scss';
+import { postPostComment } from '../../api/comments';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+type Props = {
+  postId: number;
+  commentsNumber: number;
+  setCommentsNumber: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export const NewCommentForm: React.FC<Props> = (
+  { postId, commentsNumber, setCommentsNumber },
+) => {
+  const [name, setName] = useState<string>('');
+  const [email, setEmail] = useState<string>('');
+  const [body, setBody] = useState<string>('');
+
+  const submitHandler = () => {
+    if (name === '' || email === '' || body === '') {
+      return;
+    }
+
+    const createdAt = (new Date()).toISOString();
+    const updatedAt = createdAt;
+
+    const newComment = {
+      body,
+      createdAt,
+      email,
+      id: commentsNumber + 1,
```

Comment:
it will be not unique 
and id, and date things are added by server to db, there is no need to send it for you

Reply:
https://mate-academy.github.io/fe-students-api/comments.html
here is an example of post